### PR TITLE
Bump SBT

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Uses coursier by default.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,12 +2,11 @@ addSbtPlugin("com.localytics" % "sbt-dynamodb" % "2.0.3")
...
-resolvers += "jgit-repo" at "http://download.eclipse.org/jgit/maven"
 addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages"    % "0.6.3")
 addSbtPlugin("com.typesafe.sbt" % "sbt-site"       % "1.4.0")
 addSbtPlugin("com.47deg"        % "sbt-microsites" % "0.7.23")
 addSbtPlugin("org.tpolecat"     % "tut-plugin"     % "0.6.12")
 addSbtPlugin("com.eed3si9n"     % "sbt-unidoc"     % "0.4.2")
 addSbtPlugin("com.geirsson"     % "sbt-ci-release" % "1.2.6")
 addSbtPlugin("org.scoverage"    % "sbt-scoverage"  % "1.6.0")
-addSbtPlugin("org.scalameta"    % "sbt-scalafmt"   % "2.0.4")
+addSbtPlugin("org.scalameta"    % "sbt-scalafmt"   % "2.0.3")
```
Comment (johnduffell) [inline project/plugins.sbt]: downgrading scalafmt?

Comment (regiskuckaertz) [reply project/plugins.sbt]: woops, no that's a mistake 😅

**Comment by regiskuckaertz** [comment]
That's about the only useful information I could glean from sbt inexistent release notes 😞

**Comment by johnduffell** [comment]
haha, well in their defence it's a RC version so I would expect less official information available!
Although the fact it's RC it would be nice the description on this PR to acknowledge why a RC version is appropriate in this case

**Pull request merged**